write-fonts/src/tables/glyf.rs

```diff
@@ -0,0 +1,298 @@
+//! The [glyf (Glyph Data)](https://docs.microsoft.com/en-us/typography/opentype/spec/glyf) table
+
+use kurbo::{BezPath, Shape};
+
+use read_fonts::tables::glyf::{CurvePoint, SimpleGlyphFlags};
+
+use crate::FontWrite;
+
+/// A single quadratic bezier contour
```

Comment:
Looking at the definition I think you mean it won't contain any curve of order 3 or higher. That is, there will be no sequence of more than 2 off-curve points between on-curves. The descritpion made me think it would exclusively contain quads but you appear to allow lines as well.

Perhaps the comment could be made more specific?